core/container/externalbuilder/copy_test.go

```diff
@@ -45,51 +58,77 @@ var _ = Describe("copy", func() {
 		os.RemoveAll(destRootDir)
 	})
 
-	When("dest dir does not exist", func() {
-		BeforeEach(func() {
-			destRootDir, err = ioutil.TempDir("", "dest-")
-			Expect(err).NotTo(HaveOccurred())
-			err = os.RemoveAll(destRootDir)
+	It("Copies all files and subdirectories", func() {
+		err = CopyDir(logger, srcRootDir, destRootDir)
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err := os.Stat(destRootDir)
+		Expect(err).NotTo(HaveOccurred())
```

Comment:
What is this attempting to accomplish? If it's trying to assert the directory exists, `Expect(destRootDir).To(BeADirectory())` would be much clearer.